spec/collections/samples/sample_ontology_link.yaml

```diff
@@ -0,0 +1,47 @@
+name: sample_ontology_link
+type: edge
+schema:
+  "$schema": http://json-schema.org/draft-07/schema#
+  type: object
+  required: [_from, _to, created]
+  description: The _from sample node is associated with the _to taxon.
```

Comment:
This would be better going somewhere else. It isn't necessary to have a description for every attribute in your schema. You can also consider putting in a `title`, `example`s, and validation for your attributes, if you want to do that.

Have a look at these schemas for examples:
- https://github.com/kbase/relation_engine/blob/develop/spec/collections/silva/silva_taxon.yaml
- https://github.com/kbase/relation_engine/blob/develop/spec/datasets/djornl/definitions.yaml

There's a great guide to JSON schema here if you want more info: https://json-schema.org/understanding-json-schema/

Reply:
I agree that you only need the property descriptions, but no biggie